src/ray/protobuf/common.proto

```diff
@@ -214,7 +214,7 @@ message RayErrorInfo {
     string error_message = 4;
   }
   // The type of error that caused the exception.
-  optional ErrorType error_type = 11;
+  ErrorType error_type = 11;
```

Comment:
Are you going to follow up with the error detail in the next PR?

Reply:
Yeah, otherwise the PR is rather large. Added PR name to reflect the follow up.